project/Dependencies.scala

```diff
@@ -150,7 +150,7 @@ object Dependency {
     val Mockito = "1.10.19"
     val ScalaCheck = "1.13.5"
     val ScalaTest = "3.0.4"
-    val UsiTestUtil = "0.1.12"
+    val UsiTestUtil = "0.1.30-a8aeddb-SNAPSHOT" // TODO - merge https://github.com/mesosphere/usi/pull/135
```

Comment:
The patch landed. We should be able to change this to `0.1.31` once the master build of USI is through.